.pre-commit-config.yaml

```diff
@@ -3,3 +3,8 @@ repos:
     rev: 20.8b1
     hooks:
     -   id: black
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+    -   id: flake8
+        exclude: scanpy/tests/
```

Comment:
Can we just have exclude be defined in one place?

Reply:
No, unfortunately not. pre-commit ignores excludes in the .flake8 file. 

https://github.com/pre-commit/pre-commit-hooks/issues/112